tests/integration/test_compatibility.py

```diff
@@ -329,7 +348,7 @@ def test_agg_count_distinct_no_group_by():
 
 
 @pytest.mark.skip(
-    reason="WIP DataFusion - https://github.com/dask-contrib/dask-sql/issues/532"
+    reason="conflicting aggregation functions: [('count', 'c'), ('count', 'c')]"
```

Comment:
Out of curiosity is this coming from 
```
COUNT(c) AS c_c,
COUNT(DISTINCT c) AS cd_c
```
where the optimization creates a `count(c)` for the latter case as well?